packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -64,4 +64,14 @@
       cursor: pointer;
     }
   }
+
+  .profile-container {
+    bottom: 0;
+    height: auto;
+    left: 0;
+    position: absolute;
+    text-align: left;
+    width: inherit;
+    z-index: 200;
```

Comment:
why is it 200? the general rule is ordering the z-index based on current stack context or local context: https://developer.mozilla.org/en-US/docs/Web/CSS/z-index